### ISSUE TITLE
Loot Doesn't Recognize My Skyrim Game

### ISSUE THREAD

**Issue by nawdew** [loot/loot#1175]
Good Day.

I recently updated Loot, Sunday Jan 5, 202,=0, and now, Loot doesn't recognize that I have Skyrim on my PC. The pervious version of Loot worked fine and I did not have to deal with codes, paths, etc. I get this error message:

Error: Game-specific settings could not be initialised. Cannot set the current game: the game with folder "Skyrim" is not installed.

This is not true. I need some help because I like and enjoy having a mod load order optimizing tool. The help I need must be in layman's terms. I am not a computer programmer or have any real understanding about codes, so any help must be as simple as possible.

However, I do know how to get around the registry which, I cannot find Loot in my registry except for one entry. Here as well, I must have very simple instructions. What is also odd is that the Loot icon does not show up in my installed programs list. It is a generic icon.

Please, I really could use some help here.

Sincerely;
Richard J. Woerner

**Comment by CptMcSplody** [comment]
#### [How To Report Issues Helpfully](https://loot.github.io/docs/help/How-To-Report-Issues-Helpfully.html)

> Reporting an Issue:
> It is important that if you want to report an issue, that you always include as much information as possible. In general, you should first read the above article "How To Report Issues Helpfully" - and provide us with your Debug Log "LOOTDebugLog.txt". If the Debug Log isn't provided, potential issues will only take longer to be resolved. 

**Enable** debug logging in LOOT settings and upload a full _LOOTDebugLog.txt_.

![image](https://user-images.githubusercontent.com/1944639/67115390-ff982580-f1d5-11e9-82f1-9202da269976.png)

A shortcut to the "LOOTDebugLog.txt" is provided in the LOOT hamburger menu.

![image](https://user-images.githubusercontent.com/1944639/71787574-9627f600-3011-11ea-8d93-62d9713a492f.png)

![image](https://user-images.githubusercontent.com/1944639/71787554-5a8d2c00-3011-11ea-922e-58643c58b75b.png)

**Comment by Tiflit** [comment]
Had the same issue here :
On the main screen, where you get that warning, just select Skyrim (SE in my case).
![LOOT](https://user-images.githubusercontent.com/59609757/71898129-864e0600-3126-11ea-8e36-21837aaaa2d5.png)
LOOT worked properly within MO2 after that. I am using the standalone version, not the installer. I did not manually modify the registry nor the installation files whatsoever, and my game folder in located in C:/Skyrim. I need to reselect the game like this every time I open a new instance of LOOT, first time with a clean install, and now with several mods.

Just done some more testing, and it was an MO2 issue. I had something written in the ''arguments'' line for the executable configuration. LOOT was trying to recognize a game that is not supported, in @nawdew 's case would be ''Skyrim''. Removed it, works fine.
